README-Milestone3.md

```diff
@@ -0,0 +1,20 @@
+# Milestone 3 - Invite a Participant to Your Uni
+In this section, you will ask one of your colleagues to join an existing Uni
+
+## Explore the /invitations directory
+The `Invites.md` files contains a mapping of Senders and Recipients.  
+
+## Invite Participants
+Each Sender will invite the corresponding Recipient to join one of its Unis, using the Share UI.   The Recipient will receive the invitation and join the Uni. 
+
+The Recipient will get to choose the Region in which its Node will be created (it does not have to be in the same Region as the Sender's Node).
```

Comment:
It would be helpful to outline the Advanced options just so attendees have a sense of the options that can be set.

Reply:
Not a blocker - resolving this one.